The musl libc does not define the _sysconf(2) constant _SC_LONG_BIT,
thus include a local copy of GNU libc's /usr/include/bits/confname.h

--- src/core/abi.cc	2015-10-14 10:09:57.131324977 +0200
+++ src/core/abi.cc	2015-10-14 10:15:29.040299081 +0200
@@ -8,6 +8,9 @@
 #include "config.h"
 #include "abi.h"
 #include "osutils.h"
+#if !defined(__GLIBC__)
+#include "confname.h"
+#endif
 #include <unistd.h>
 #include <stdlib.h>
 #include <dirent.h>
